src/NosCore.GameObject/Character.cs

```diff
@@ -77,18 +86,26 @@ public class Character : CharacterDto, ICharacterEntity
         private readonly IDao<MinilandDto, Guid> _minilandDao;
         private readonly IMinilandProvider _minilandProvider;
         private readonly IDao<QuicklistEntryDto, Guid> _quicklistEntriesDao;
-
         private readonly IDao<StaticBonusDto, long> _staticBonusDao;
         private readonly IDao<TitleDto, Guid> _titleDao;
-        private byte _speed;
         private readonly IDao<CharacterQuestDto, Guid> _characterQuestsDao;
+        private readonly IHpService _hpService;
+        private readonly IMpService _mpService;
+        private readonly ISpeedService _speedService;
+        private readonly IExperienceService _experienceService;
+        private readonly IJobExperienceService _jobExperienceService;
+        private readonly IHeroExperienceService _heroExperienceService;
+        private readonly IReputationService _reputationService;
+        private readonly IDignityService _dignityService;
 
         public Character(IInventoryService inventory, IExchangeProvider exchangeProvider, IItemProvider itemProvider,
             IDao<CharacterDto, long> characterDao, IDao<IItemInstanceDto?, Guid> itemInstanceDao,
             IDao<InventoryItemInstanceDto, Guid> inventoryItemInstanceDao, IDao<AccountDto, long> accountDao,
             ILogger logger, IDao<StaticBonusDto, long> staticBonusDao,
             IDao<QuicklistEntryDto, Guid> quicklistEntriesDao, IDao<MinilandDto, Guid> minilandDao,
-            IMinilandProvider minilandProvider, IDao<TitleDto, Guid> titleDao, IDao<CharacterQuestDto, Guid> characterQuestDao)
+            IMinilandProvider minilandProvider, IDao<TitleDto, Guid> titleDao, IDao<CharacterQuestDto, Guid> characterQuestDao,
+            IHpService hpService, IMpService mpService, IExperienceService experienceService, IJobExperienceService jobExperienceService, IHeroExperienceService heroExperienceService, ISpeedService speedService,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 211 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5574106)